libs/MVS/DepthMap.cpp

```diff
@@ -72,6 +72,7 @@ MDEFVAR_OPTDENSE_uint32(nMinViewsFilterAdjust, "Min Views Filter Adjust", "minim
 MDEFVAR_OPTDENSE_uint32(nMinViewsTrustPoint, "Min Views Trust Point", "min-number of views so that the point is considered for approximating the depth-maps (<2 - random initialization)", "2")
 MDEFVAR_OPTDENSE_uint32(nNumViews, "Num Views", "Number of views used for depth-map estimation (0 - all views available)", "0", "1", "4")
 MDEFVAR_OPTDENSE_bool(bFilterAdjust, "Filter Adjust", "adjust depth estimates during filtering", "1")
+MDEFVAR_OPTDENSE_bool(bFuse, "Fuse", "fuse depth maps into a point cloud", "1")
```

Comment:
if you add a command line param for fusion, no need to add it also as a INI param, simply pass it as a variable to densification fucntion

Reply:
Ok thank you. I was not sure. I was just trying to be exhaustive here.

Reply:
Ok, I'm finally not sure of what to fix here. Densification function is not taking any argument (on the contrary to Refine and Mesh functions for example). Should I remove this line?

Reply:
I was trying to suggest to add it as a param to `bool Scene::DenseReconstruction()` function, but u do as u think is better

Reply:
Well, I want to maximize the chances to see this code merged into master one day ;). I'm not totally against the parameter, I just wanted to be consistent with existing code.

Reply:
I hope I understood well and I have implemented the method parameter in this commit 813bde73b5b3ff04a149671c811a498e64611ebd so feel free to pick the one you prefer.